app-modules/authorization/tests/Feature/Filament/Resources/User/ViewUserTest.php

```diff
@@ -131,3 +131,190 @@
     expect($second->isImpersonated())->toBeTrue();
     expect(auth()->id())->toBe($second->id);
 });
+
+it('does not display the mfa_status Action for an external User', function () {
+    $user = User::factory()->create([
+        'is_external' => true,
+    ]);
```

Comment:
We have an external factory state function.

```suggestion
    $user = User::factory()->external()->create();
```